app.js

```diff
@@ -41,10 +45,21 @@ app.use( ( req , res , next ) => {
 
 // psql defined errors
 app.use( ( err , req , res , next ) => {
-  if ( err.code === '23502' || err.code === '22P02') {
+  if ( 
+    err.code === "23502" // not_null_violation
```

Comment:
Have a think about what each of these error codes mean. It would be more helpful to send back a tailored error message for each one to help the user know what has gone wrong.